### PR TITLE
make object.assign an explict dependency

### PR DESCRIPTION
**Pull request by dvehar**
@astonm pointed out that object.assign is needed by node_modules/forms
I'm able to start the app with these changes (the usage of object.assign comes from its README). The version might not be right.
What do you think? I'm curious as to why it would work for you but not me.
Did you have object.assign installed globally? I'd think so but then I think we'd have a problem when it is deployed unless we deploy in a way that is different then the README.
Maybe there is something else going on that I don't understand...

### CODE REVIEW

**Comment by makinde** [comment]
Can you run `npm -v` and `node -v` to see what version you're on? I'm on `2.14.12` and `v4.2.6` respectively. I'm guessing there's a mismatch here. Object.assign is now in the ES6 spec and should be supported by the latest version of node

**Comment by dvehar** [comment]
👍
I'm on 2.11.3 and v0.12.7
I bumped up to v4.2.6 and it worked.
I will reflect this in README

File: README.md [hunk 1]
```diff
@@ -4,9 +4,10 @@ The goals of this site are to (a) inform members about /dev/color organization a
 
 ## Dev Setup Instructions
 
-1. `npm install` to get all the dependencies
-2. copy `.env.example` to `.env` and edit as needed
-3. either `source .env` for each terminal session or set up [autoenv](https://github.com/kennethreitz/autoenv)
-4. The env file points to a shared development database. Ask @makinde, @astonm, or @dvehar to create a profile for you in the test DB so you can login. You can setup MongoDB locally if you'd like. Just point the MONGO_DEV_URI env variable to your local instance.
-5. start the app with `npm start`, `heroku local web` to simulate Heroku, or install nodemon (`npm install -g nodemon`) and just run `nodemon` to have the server restart automatically when you change files
-6. before committing, run `jshint .` to check that your javascript is all good (`npm install -g jshint` if you need it)
+1. It is suggested to have npm version 2.14.12 and node version v4.2.6 installed
```
Comment (makinde) [inline README.md]: How did you do the upgrade? Is there a command to run? If so, perhaps put that directly in here.

File: README.md [hunk 2]
```diff
@@ -4,9 +4,10 @@ The goals of this site are to (a) inform members about /dev/color organization a
 
 ## Dev Setup Instructions
 
-1. `npm install` to get all the dependencies
-2. copy `.env.example` to `.env` and edit as needed
-3. either `source .env` for each terminal session or set up [autoenv](https://github.com/kennethreitz/autoenv)
-4. The env file points to a shared development database. Ask @makinde, @astonm, or @dvehar to create a profile for you in the test DB so you can login. You can setup MongoDB locally if you'd like. Just point the MONGO_DEV_URI env variable to your local instance.
```
Comment (makinde) [inline README.md]: This part got lost in the merge: `Ask @makinde, @astonm, or @dvehar to create a profile for you in the test DB so you can login.`

**Comment by makinde** [comment]
You're fighting the good fight! These tweaks will make it easier for more folks to hop in, and your first pull request will land soon! Almost there.

File: README.md (hunk 2 above)
Comment (dvehar) [inline README.md]: 👍

File: README.md (hunk 1 above)
Comment (dvehar) [inline README.md]: 👍

**Pull request merged**